net/DevExtreme.AspNet.Data/Helpers/EFSorting.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace DevExtreme.AspNet.Data.Helpers {
+
+    static class EFSorting {
+
+        static readonly ICollection<Type> LIKELY_KEY_TYPES = new[] {
```

Comment:
Well, yes. Also I think that splitting types into 2 categories isn't necessary. Instead I can use a single array with types ordered from "likely"-keys to "unlikely"-keys.